### ISSUE TITLE
curiousity killed the cat

### ISSUE THREAD

**Issue by andreineculau** [zkochan/ignorable#2]
Why do you say it's unsafe to ignore test.js and example.js ?

And why do you say it's safe to ignore AUTHORS files? it's not actually. both AUTHORS and NOTICE :s

**Comment by zkochan** [comment]
my initial assumption was that it is unsafe to ignore any code file and safe to remove docs/markdowns and other plain text files. I was warned that removing license is not good from the legal perspective.

**Comment by andreineculau** [comment]
hmm, it's peculiar though. READMEs are safe to delete, but not `example.js`. Same for the `test` folder but not `test.js`..

regarding AUTHORS and NOTICE - they follow in the legal bucket, as the LICENSE may say that all rights are reserved to those mentioned in the AUTHORS file, and/or that reuse of the work need to embed the NOTICE

**Comment by zkochan** [comment]
yes, the test folder should be treated same way as `test.js`. That is a mistake.

Regarding AUTHORS and NOTICE. The LICENSE file can say anything.

And also, what safe, what is unsafe is super-subjective. I'd prefer removing the distinction and having just the unsafe set.
